client/src/components/leftSidebar/infoMenu.tsx

```diff
@@ -35,7 +35,7 @@ const InformationMenu = React.memo<Props>((props): JSX.Element => {
           <MenuItem
             icon={<Icon icon={CXGIconNames.ABOUT} />}
             popoverProps={{ openOnTargetFocus: false }}
-            text="About cellxgene"
+            text="About CellxGene"
```

Comment:
```suggestion
            text="About CELLxGENE"
```

@seve can you sanity check this change please?

Reply:
should it be CZ CELLxGENE?

Reply:
CZ CELLxGENE Discover.

Reply:
Updated!